spec/requests/api_keys_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe "ApiKeys", type: :request do
+  describe "GET /api_keys" do
+    it "works! (now write some real specs)" do
```

Comment:
As it seems only the api_keys has the requests and routes spec, should I just remove them to conform to the rest of the app?

Basically the /api_keys is such a simple piece that this spec would serve just fine as is.